tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -72,10 +68,62 @@ def test_createami(region, os, instance, request, pcluster_config_reader, vpc_st
         any("packages.chef.io".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
     ).is_false()
     assert_that(
-        any("Thank you for installing Chef".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+        any(
+            "Thank you for installing Cinc Client".lower() in s.lower()
+            for s in pcluster_createami_result.stdout.split("\n")
+        )
     ).is_true()
     assert_that(
-        any("Starting Chef Client".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+        any("Starting Cinc Client".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
     ).is_true()
 
     assert_that(pcluster_createami_result.stdout).does_not_contain("No custom AMI created")
+
+
+@pytest.mark.dimensions("us-west-2", "c5.xlarge", "centos7", "*")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "ubuntu1804", "*")
+@pytest.mark.dimensions("eu-west-1", "m6g.xlarge", "alinux2", "*")
+def test_createami_post_install(
+    region, os, instance, test_datadir, request, pcluster_config_reader, vpc_stack, architecture
+):
+    """Test post install script and base AMI is ParallelCluster AMI"""
+    cluster_config = pcluster_config_reader()
+
+    # Get ParallelCluster AMI as base AMI
+    base_ami = retrieve_latest_ami(region, os, ami_type="pcluster", architecture=architecture)
+
+    # Networking
+    vpc_id = vpc_stack.cfn_outputs["VpcId"]
+    networking_args = ["--vpc-id", vpc_id, "--subnet-id", vpc_stack.cfn_outputs["PublicSubnetId"]]
+
+    # Custom Cookbook
+    custom_cookbook = request.config.getoption("createami_custom_chef_cookbook")
+    custom_cookbook_args = [] if not custom_cookbook else ["-cc", custom_cookbook]
+
+    # Instance type
+    pcluster_version_result = run_command(["pcluster", "version"])
+    instance_args = (
+        [] if version.parse(pcluster_version_result.stdout.strip()) < version.parse("2.4.1") else ["-i", instance]
+    )
+
+    # Post install script
+    post_install_script = (
```

Comment:
To avoid to [repeat yourself](https://realpython.com/lessons/dry-principle/) with `"file://"` and `str(test_datadir...)`  you could rewrite it with:
```python
post_install_script_file = "post_install_ubuntu.sh" if os in ["ubuntu1804", "ubuntu1604"] else "post_install.sh"
post_install_script = "file://{0}".format(test_datadir / post_install_script_file)
```

Reply:
Done